troposphere/static/js/components/modals/instance/image/components/Tags.react.js

```diff
@@ -56,13 +104,15 @@ define(function (require) {
             <TagMultiSelect
               models={tags}
               activeModels={imageTags}
-              onModelAdded={this.props.onTagAdded}
+              onCreateNewTag={this.onCreateNewTag}
+							onModelAdded={this.props.onTagAdded}
               onModelRemoved={this.props.onTagRemoved}
               onModelCreated={this.props.onTagCreated}
               onQueryChange={this.onQueryChange}
               width={"100%"}
               placeholderText="Search by tag name..."
               />
+						{tagCreateForm}
           </div>
         </div>
       );
```

Comment:
My commit was to `jetstream` - just to clarify (removes the mix of tabs & spaces).